website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -163,82 +179,81 @@ The command uninstalls the release and removes all Kubernetes resources associat
 | `storageClass.mountOptions`         | Mount options used by Persistent Volumes                             | `{}`               |
 | `storageClass.parameters`           | Storage Class parameters                                             | `{}`               |
 
-#### Auto-update chart's image
+#### Auto update image
 
 | Name                   | Description                                   | Value         |
 | ---------------------- | --------------------------------------------- | ------------- |
 | `autoupdate.enabled`   | Enable auto update Helm chart's image         | `true`        |
 | `autoupdate.scheduler` | Schedule time to run cron job to update image | `"0 * * * *"` |
 
-Specify each parameter using `--set key=value[,key=value]` argument to helm install. For example:
+### Configure parameters
```

Comment:
there shouldn't be a separate section again. Why can't we put this right before the table of parameters under the main Configure heading followed by the parameters list?

we should spread this content properly so there is a flow and there is no break in the information leading to confusion

Reply:
I think the parameter table gives a view of all the parameters which the users can configure, and then the description below shows how to configure them. I felt adding a separate section will help readability.

Reply:
As discussed, I'll add a few lines of explanation followed by the table and keep the examples below the table.